blockmanager.go

```diff
@@ -43,6 +43,10 @@ const (
 	// maxCFCheckptsPerQuery is the maximum number of filter header
 	// checkpoints we can query for within a single message over the wire.
 	maxCFCheckptsPerQuery = wire.MaxCFHeadersPerMsg / wire.CFCheckptInterval
+
+	// fType is the filter type used.
```

Comment:
Not sure if the global variable is needed given the existence of: `wire.GCSFilterRegular`

Reply:
I made it global so that the two subsystems sideloader and blockmanager would use came filter type. 

Although I do not know if it would be better to make this part of neutrino config, the users define the filter type to use right? If it is unsupported meutrino gives an error message. 

But making it a constant global variable seems similary opposite to its previous state as a local variable.

Reply:
Why's the global needed. We already have ` wire.GCSFilterRegular`, and _existing_ global variable.

Reply:
What happens in the future if we want to make neutrino use a different type? Do we go to the sideloader then change the filter type from `wire.GCSFilterRegular` to  say `wire.NewFilterType` then go ahead to do the same for the blockmanger fetching cfheader from p2p source?

Reply:
I think that would manifest as a diff implementation of the `HeaderWriter` interface. Header validation would be the same, only our behavior when fetching a filter changes. FWIW, we haven't added a filter in several years.